[feature] support OpenBSD

trombik opened the pull request:
fixes #14

ruby version is hard-coded and should be fixed by writing `ansible-role-ruby`

Commit:
do NOT use `enabled: true` in handlers

in most cases, you set `enbled: true` in a service task in tasks. it
might look same when you set `enabled: true` in BOTH a service task and
a service handler, but it would bites you when you have `arguments` in a
service task.

`arguments` is not implemented in all platforms. it is in OpenBSD. when
both the service task and the handler has `enabled: true`, but only
one of them has `arguments`, then, `idempotency_test` will mysteriously
fail because the one with `arguments` modifies `rc.conf.local` and the
other as well.

this happens when:

* the platform is OpenBSD
* a service in task and a service in handler has `enabled: true`
* only one of them has `arguments`
* `arguments` is not identical to the default

Commit:
support OpenBSD

fixes #14

Commit:
QA

Commit:
merge master

Commit:
follow changes in master

Commit:
update README

Commit:
Merge branch 'master' into issue_14

mitsururike commented on tasks/install-OpenBSD.yml:
```diff
@@ -0,0 +1,49 @@
+- name: Create _fluentd user
+  user:
+    name: _fluentd
+    comment: Fluend daemon
```
s/Fluend/Fluentd/

mitsururike commented on vars/OpenBSD.yml:
```diff
@@ -0,0 +1,13 @@
+__fluentd_package_name: rubygem-fluentd
+__fluentd_service_name: fluentd
+__fluentd_config_dir: "/etc/fluentd"
+__fluentd_config_file: "{{ __fluentd_config_dir }}/fluent.conf"
```
s/__fluentd_config_dir/fluentd_config_dir/ or users must modify fluentd_config_file when fluentd_config_dir is set.

Commit:
typo

https://github.com/reallyenglish/ansible-role-fluentd/pull/41#pullrequestreview-39638234

Commit:
use user-defindable variable

Commit:
apply same fix: 58924778

mitsururike commented on README.md:
```diff
@@ -88,6 +88,23 @@ Note that although the role creates `fluentd_log_dir`, you need to configure
+| `__fluentd_package_name` | `rubygem-fluentd` |
+| `__fluentd_service_name` | `fluentd` |
+| `__fluentd_config_dir` | `/etc/fluentd` |
+| `__fluentd_config_file` | `{{ __fluentd_config_dir }}/fluent.conf` |
```
s/__fluentd_config_dir/fluentd_config_dir/

Commit:
sync
